Week3/homework/calculateDogAge.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+function calculateDogAge(puppysAge) {
+    const dogYears = 7 * puppysAge;
+    return console.log(`Your doggie is${dogYears} years old in dog years!`);
```

Comment:
The question asks to return a string here.

See if you can update the code to fix this. Hint: Move the `console.log` outside this function.

Reply:
Thank you for the feed back I fixed.